### DISCUSSION TITLE
Added Feature Toggle

### DISCUSSION THREAD

**Discussion by ItWasEnder** [GriefPrevention/GriefPrevention#1159] [General]
Maybe an enabled section for all the features of grief prevention. GriefPrevention keeps messing with player punishments when I do not want it. I think griefprevention is a really nice claiming plugin but nothing else.

**Comment by RoboMWM** [comment]
part of the plan for the road to v20

**Reply by ItWasEnder** [reply]
What does the timeline of that look like?

**Reply by RoboMWM** [reply]
I created a discussion about it a couple hours ago #1162 and I've discussed a few times before in an older v20 discussion in #63
